src/main/database/entities/DriveFile.ts

```diff
@@ -0,0 +1,99 @@
+import { DatabaseCollectionAdapter } from '../adapters/base';
+
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+import { AppDataSource } from '../data-source';
+
+@Entity()
+export class DriveFile {
+  @Column({ nullable: false })
+  fileId!: string;
+
+  @Column({ nullable: false })
+  id!: number;
+
+  @PrimaryColumn({
+    nullable: false,
+  })
+  uuid!: string;
+
+  @Column({ nullable: false })
+  type!: string;
+
+  @Column({ nullable: false })
+  size!: number;
+
+  @Column({ nullable: false })
+  bucket!: string;
+
+  @Column({ nullable: false })
+  folderId!: number;
+
+  @Column({ nullable: true })
+  folderUuid?: string;
+
+  @Column({ nullable: false })
+  userId!: number;
+
+  @Column({ nullable: true })
+  modificationTime?: string;
```

Comment:
`modificationTime` is not deprecated?

Reply:
I believe is the same as updatedAt since the dates I receive are the same, can you confirm it @sg-gs ?

Reply:
I confirm that updatedAt and modificationTime are different fields, modificationTime is not deprecated